app/frontend/svgs/filter.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
For consistency's sake, TailwindUI uses the font pack below and I would copy and paste the raw values:

https://github.com/refactoringui/heroicons

I really did how it looks now, so I'm 100% for leaving this as is, but I'd be curios if that pack has something similar so we can be consistent.

Reply:
got it. I wasn't aware about heroicons. will refactor.